e2e/config.js

```diff
@@ -0,0 +1,144 @@
+        response: "1\n",
+      },
+      {
+        expectedPrompt: /Do you wish to use template extensibility?/i,
+        response: "2\n",
+      },
+      {
+        expectedPrompt: /What is the name of your Project?/i,
+        response: "scaffold-pwa\n",
+      },
+      {
+        expectedPrompt: /What is the URL for your Commerce Cloud instance?/i,
+        response: "https://zzte-053.dx.commercecloud.salesforce.com\n",
+      },
+      {
+        expectedPrompt: /What is your SLAS Client ID?/i,
+        response: "1d763261-6522-4913-9d52-5d947d3b94c4\n",
+      },
+      {
+        expectedPrompt: /What is your Site ID in Business Manager?/i,
+        response: "RefArch\n",
+      },
+      {
+        expectedPrompt:
+          /What is your Commerce API organization ID in Business Manager?/i,
+        response: "f_ecom_zzte_053\n",
+      },
+      {
+        expectedPrompt:
+          /What is your Commerce API short code in Business Manager?/i,
+        response: "kv7kzm78\n",
+      },
+    ],
+    "retail-app-no-ext": [
+      {
+        expectedPrompt: /Choose a project preset to get started:/i,
+        response: "1\n",
+      },
+      {
+        expectedPrompt: /Do you wish to use template extensibility?/i,
+        response: "1\n",
+      },
+      {
+        expectedPrompt: /What is the name of your Project?/i,
+        response: "scaffold-pwa\n",
+      },
+      {
+        expectedPrompt: /What is the URL for your Commerce Cloud instance?/i,
+        response: "https://zzte-053.dx.commercecloud.salesforce.com\n",
+      },
+      {
+        expectedPrompt: /What is your SLAS Client ID?/i,
+        response: "1d763261-6522-4913-9d52-5d947d3b94c4\n",
+      },
+      {
+        expectedPrompt: /What is your Site ID in Business Manager?/i,
+        response: "RefArch\n",
+      },
+      {
+        expectedPrompt:
+          /What is your Commerce API organization ID in Business Manager?/i,
+        response: "f_ecom_zzte_053\n",
+      },
+      {
+        expectedPrompt:
+          /What is your Commerce API short code in Business Manager?/i,
+        response: "kv7kzm78\n",
+      },
+    ],
+  },
+  EXPECTED_GENERATED_ARTIFACTS: {
+    "retail-app-demo": [
+      ".eslintignore",
+      ".eslintrc.js",
+      ".prettierrc.yaml",
+      "babel.config.js",
+      "config",
+      "node_modules",
+      "overrides",
+      "package-lock.json",
+      "package.json",
+      "translations",
+      "worker",
+    ],
+    "retail-app-ext": [
+      ".eslintignore",
+      ".eslintrc.js",
+      ".prettierrc.yaml",
+      "babel.config.js",
+      "config",
+      "node_modules",
+      "overrides",
+      "package-lock.json",
+      "package.json",
+      "translations",
+      "worker",
+    ],
+    "retail-app-no-ext": [
+      ".eslintignore",
+      ".eslintrc.js",
+      ".prettierignore",
+      ".prettierrc.yaml",
+      "CHANGELOG.md",
+      "LICENSE",
+      "README.md",
+      "app",
```

Comment:
so we only check at the top-level dimension of the filesystem? Is it ok that we don't drill into `app` and validate there?

Reply:
We can drill in further. For now I've just limited the scope to check top-level files and for extensibility we dig into package.json to confirm `ccExtensibility` is defined.
We need consensus upon how deep into the repo do we want to dig in.

Reply:
Yeah, top level is probably fine for now. I like your idea of generating checksums, but obviously we don't have any interest in that unless it could be fully automated

Reply:
If I recall the previous discussion on this correctly, we wanted some way to verify the configs are being correctly set on generation, correct? 

The checksum helps with that but until then do we want a separate follow up story for checking the contents of config/default.js and ensuring they match what was entered in the prompt?

Reply:
I can create a follow-up ticket to collect all the assertions we want to make on the generated projects and add them to the scripts later depending on how much automation is possible.